addons/l10n_fi/data/account_account_template_data.xml

```diff
@@ -0,0 +1,5403 @@
+        <field name="code">116105</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Lis&#228;poistot</field>
+    </record>
+    <record model="account.account.template" id="a116106">
+        <field name="code">116106</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Poistoeron muutos</field>
+    </record>
+    <record model="account.account.template" id="a116107">
+        <field name="code">116107</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Poistoero</field>
+    </record>
+    <record model="account.account.template" id="a116200">
+        <field name="code">116200</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Televiestint&#228;- ,audio- ,video, muut ICT</field>
+    </record>
+    <record model="account.account.template" id="a116201">
+        <field name="code">116201</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Lis&#228;ykset</field>
+    </record>
+    <record model="account.account.template" id="a116202">
+        <field name="code">116202</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">V&#228;hennykset</field>
+    </record>
+    <record model="account.account.template" id="a116203">
+        <field name="code">116203</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Katettu j&#228;lleenhankintavarauksella</field>
+    </record>
+    <record model="account.account.template" id="a116204">
+        <field name="code">116204</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">
```

Comment:
Do we need all 700+ accounts?  It seems they are generic enough, but always good if you could check if we can skip some.  Also, csv is better for loading in large quantities.

Reply:
I will consult our accounting specialist if we could cut off some accounts or possibly add them as child-CoA's.

Reply:
There's nothing really that could be left out. It's a generic CoA and yes, there are quite a few accounts, but there's no compact version available.